cmd/nanotube/main_test.go

```diff
@@ -46,9 +46,17 @@ func setup(t *testing.T) {
 
 	cfgPath := filepath.Join(fixturesPath, "config.toml")
 
-	cfg, clusters, rules, rewrites, ms, _, err := loadBuildRegister(cfgPath, lg)
+	cfg, clustersConf, rulesConf, rewritesConf, _, err := readConfigs(cfgPath)
 	if err != nil {
-		t.Fatalf("failed to build config: %v", err)
+		t.Fatalf("error reading and compiling config: %v", err)
+	}
+	if err != nil {
```

Comment:
Checking for the same errors twice

Reply:
It's actually a remnant of the logger build) Thanks!